barcode/codex.py

```diff
@@ -61,7 +61,7 @@ def get_fullcode(self):
         return self.code
 
     def calculate_checksum(self):
-        check = sum(code39.MAP[x][0] for x in self.code) % 43
+        check = sum([code39.MAP[x][0] for x in self.code]) % 43
```

Comment:
List comprehension seems redundant here.

Reply:
Completely agree, not sure how that line got changed.

Reply:
No I didn't see an example of it being added twice, but if anyone called the render function twice then it would.
Agreed on list comprehension